doc/user_guide/model.rst

```diff
@@ -1051,6 +1051,21 @@ reaching the end of the dataset.
     :py:meth:`~.model.BaseModel.multifit` to perform the fit over the entire
     spectrum image.
 
+A common problem is that the fitting fails going from the end of one row to the
```

Comment:
This would make more sense to the reader if the other "scanning pattern" was also described. Also it would be good to add a ``..versionadded::`` directive.